crates/torii/indexer/src/processors/event_message.rs

```diff
@@ -28,6 +32,23 @@ where
         true
     }
 
+    fn task_priority(&self) -> usize {
```

Comment:
```suggestion
    fn task_priority(&self) -> TaskPriority {
```
To be consistent (more are missing, just mentioned this one).